x/pairing/keeper/msg_server_relay_payment.go

```diff
@@ -289,16 +261,19 @@ func (k msgServer) RelayPayment(goCtx context.Context, msg *types.MsgRelayPaymen
 		}
 
 		details["relayNumber"] = strconv.FormatUint(relay.RelayNum, 10)
-		// differentiate between different relays by providing the index in the keys
-		successDetails := appendRelayPaymentDetailsToEvent(details, uint64(relayIdx))
-		// calling the same event repeatedly within a transaction just appends the new keys to the event
-		utils.LogLavaEvent(ctx, logger, types.RelayPaymentEventName, successDetails, "New Proof Of Work Was Accepted")
 
-		err = k.chargeComputeUnitsToProjectAndSubscription(ctx, clientAddr, relay)
+		cuAfterQos := uint64(reward.Quo(coinsPerCu).TruncateInt64())
+		err = k.chargeComputeUnitsToProjectAndSubscription(ctx, clientAddr, relay, cuAfterQos)
 		if err != nil {
 			return nil, utils.LavaFormatError("Failed charging CU to project and subscription", err)
 		}
 
+		details["tracked_cu_added"] = strconv.FormatUint(cuAfterQos, 10)
```

Comment:
use the existing "rewardedCU" instead of tracked_cu_added

Reply:
done. see https://github.com/lavanet/lava/commit/723e53e095475d3d350262fc2e363f81f51dd938